linux_os/guide/services/mail/postfix_client/postfix_client_configure_mail_alias_postmaster/ansible/shared.yml

```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_all
+
```

Comment:
The fail seems to be legitimate:

```
183/246 Test #184: rhel8-ansible-assert-playbooks-schema ....................................***Failed    6.73 sec
Error processing /__w/content/content/build/rhel8/playbooks/all/postfix_client_configure_mail_alias_postmaster.yml: Did not found key(s) in comments: complexity, disruption, strategy, reboot
```

Please could you add the headers, for example:

```
# reboot = false
# strategy = restrict
# complexity = low
# disruption = low
```